packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.kt

```diff
@@ -108,6 +108,14 @@ class UpdatesDevLauncherController(
     launcher = null
   }
 
+  override fun getRuntimeVersion(context: Context): String? {
+    return updatesConfiguration?.getRuntimeVersion()
+  }
+
+  override fun getUpdateUrl(context: Context): String? {
+    return updatesConfiguration?.updateUrl?.toString()
```

Comment:
```suggestion
  override fun getUpdateUrl(context: Context): Uri? {
    return updatesConfiguration?.updateUrl
```

nit: maybe return Uri as aligned with ios implementation?